srtcore/sync.h

```diff
@@ -21,9 +28,37 @@
 #include <thread>
 #include <mutex>
 #include <condition_variable>
+#define SRT_SYNC_CLOCK SRT_SYNC_CLOCK_STDCXX_STEADY
+#define SRT_SYNC_CLOCK_STR "STDCXX_STEADY"
 #else
 #include <pthread.h>
+
+// Defile clock type to use
+#ifdef IA32
+#define SRT_SYNC_CLOCK SRT_SYNC_CLOCK_IA32_RDTSC
+#define SRT_SYNC_CLOCK_STR "IA32_RDTSC"
```

Comment:
As you already have symbols with assigned subsequent integer numbers, I'd suggest to rather create a static array with clock names, placed in the same file where the log displaying this value.